hydrus/resources.py

```diff
@@ -106,6 +106,13 @@ def get(self, id_: str, path: str) -> Response:
             # Get path of the collection-class
             class_path = item_class.path
             is_collection = True
+            # To check if it's a request to GET a single instance from Collection
+            params = request.args.to_dict()
+            member_id = params.get('instances', None)
+            if member_id:
```

Comment:
please be explicit with checks: `if member_id is not None`.
Also it is better to be consistent, below you correctly wrote `params.get('instances'`, why in here we have `...get('instances', None)`. `None` is a default so it should not be expressed.

Reply:
Sure. changes made